include/mbedtls/md.h

```diff
@@ -296,8 +298,9 @@ int mbedtls_md_update( mbedtls_md_context_t *ctx, const unsigned char *input, si
  * \param ctx       The generic message-digest context.
  * \param output    The buffer for the generic message-digest checksum result.
  *
- * \returns         \c 0 on success, or #MBEDTLS_ERR_MD_BAD_INPUT_DATA if
- *                  parameter verification fails.
+ * \returns         \c 0 on success.
+ * \returns         #MBEDTLS_ERR_MD_BAD_INPUT_DATA if parameter verification
```

Comment:
Suggestion: Use past tense here as in the rest of the documentation: `if [the] parameter verification failed`.

Reply:
changed phrasing to "on parameter-verification failure" to standardize.